recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ogdf" %}
+{% set version = "2020.02" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/catalpa-202002.tar.gz
+  sha256: 32f3f943675e55d1a23e62293d016dc1644b1176271d60f41788ade60c2d3fc5
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libOGDF.a  # [unix]
```

Comment:
Please do not build static libraries: see [CFEP-18](https://github.com/conda-forge/cfep/blob/master/cfep-18.md)